components/shared/README.md

```diff
@@ -0,0 +1,40 @@
+# browserd/shared
+
+Represents some functionality that is shared between the consumer and provider 🤕☁✨
+
+[![Build Status](https://dev.azure.com/bengreenier/browserd/_apis/build/status/shared?branchName=master)](https://dev.azure.com/bengreenier/browserd/_build/latest?definitionId=12&branchName=master)[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bengreenier_browserd&metric=alert_status)](https://sonarcloud.io/dashboard?id=bengreenier_browserd)
```

Comment:
let's drop the quality gate badges from these sub-readmes, and just have the one at the root